libs/langchain/langchain/memory/chat_message_histories/sql.py

```diff
@@ -49,44 +87,51 @@ def __init__(
         session_id: str,
         connection_string: str,
         table_name: str = "message_store",
+        session_id_field_name: str = "session_id",
+        custom_message_converter: Optional[BaseMessageConverter] = None,
     ):
         self.table_name = table_name
```

Comment:
i dont think we use table name any more right? lets remove it

Reply:
Removed the field, but I think we need to leave it in the constructor arguments, since it is passed to DefaultMessageConverter if no custom converter is passed. Do you think it is worth to leave a comment about this?

Reply:
nah i think fine